src/stats/archive/loader.rs

```diff
@@ -0,0 +1,93 @@
+use csv;
+use jormungandr_lib::interfaces::BlockDate;
+use serde::Deserialize;
+use std::{ffi::OsStr, fmt, path::Path};
+use thiserror::Error;
+
+#[derive(Debug, Deserialize)]
+pub struct ArchiverRecord {
+    pub fragment_id: String,
+    pub caster: String,
+    pub proposal: u32,
+    #[serde(deserialize_with = "deserialize_block_date_from_float")]
+    pub time: BlockDate,
+    pub choice: u8,
+    pub raw_fragment: String,
+}
+
+use serde::de::Visitor;
+use serde::Deserializer;
+
+pub fn deserialize_block_date_from_float<'de, D>(deserializer: D) -> Result<BlockDate, D::Error>
+where
+    D: Deserializer<'de>,
+{
+    struct VoteOptionsDeserializer();
+
+    impl<'de> Visitor<'de> for VoteOptionsDeserializer {
+        type Value = BlockDate;
+
+        fn expecting(&self, formatter: &mut fmt::Formatter) -> fmt::Result {
+            formatter.write_str("float in format {epoch}.{slod_id}")
+        }
+
+        fn visit_f64<E>(self, value: f64) -> Result<BlockDate, E>
+        where
+            E: serde::de::Error,
+        {
+            Ok(value.to_string().parse().unwrap())
```

Comment:
this should return an error instead of panicking, and btw not sure serializing blockdate as float is a good idea